manifests/config/exporter/patch/patch-kind.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kepler-exporter
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - name: kepler-exporter
+      volumes:
+      - name: proc
+        hostPath:
+          path: /proc-host
```

Comment:
@sunya-ch could you please help me understand why we are patching this for kind cluster?